packages/aws-cdk-lib/region-info/lib/default.ts

```diff
@@ -1,12 +1,15 @@
 /**
  * Provides default values for certain regional information points.
+ * @deprecated - Service principals are now globally `<SERVICE>.amazonaws.com`
  */
 export class Default {
 
   /**
    * The default value for a VPC Endpoint Service name prefix, useful if you do
    * not have a synthesize-time region literal available (all you have is
    * `{ "Ref": "AWS::Region" }`)
+   *
+   * @deprecated - Use VpceEndpointService.VPC_ENDPOINT_SERVICE_NAME_PREFIX instead
    */
   public static readonly VPC_ENDPOINT_SERVICE_NAME_PREFIX = 'com.amazonaws.vpce';
```

Comment:
You gave it a different name above, it's just `DEFAULT_PREFIX` now.

Reply:
OOOOOPS. Missed updating this.